packages/e2e/cypress/e2e/settings/invites.cy.ts

```diff
@@ -25,13 +25,13 @@ describe('Settings - Invites', () => {
             });
         cy.get('[data-cy="welcome-user"]').should('be.visible');
         cy.contains('Join your team').click();
-        cy.findByLabelText('First name *').type('Mary');
-        cy.findByLabelText('Last name *').type('Green');
-        cy.findByLabelText('Email address *')
+        cy.findByPlaceholderText('Your first name').type('Mary');
+        cy.findByPlaceholderText('Your last name').type('Green');
+        cy.get('[data-cy="email-address-input"]')
```

Comment:
can you also find this by placeholder or role or relevant detail?

Reply:
tried this with contains but it wouldn't recognize that the input was disabled
this was the only way to get the actual input and not the label